Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi9/go-toolset:1.19.13-4.1697647145 as builder
+FROM docker.io/library/golang:1.20@sha256:098d628490c97d4419ed44a23d893f37b764f3bea06e0827183e8af4120e19be as builder
```

Comment:
There is a chance mentioning this image here may get rate limited https://www.docker.com/increase-rate-limits/

Reply:
Yes, we are probably not going to be using that image, it's there to help test this change until we find a way to use an image originating from Red Hat-approved repositories.